src/main/java/ua/realestate/management/entity/User.java

```diff
@@ -0,0 +1,70 @@
+package ua.realestate.management.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import javax.persistence.*;
+import java.util.Collection;
+
+
+@Data
+@Entity
+@Table(name = "users")
+@NoArgsConstructor
+
+public class User implements UserDetails {
+
+    @Id
+    @GeneratedValue(strategy= GenerationType.AUTO)
+    private Integer id;
+
+    private String name;
+    private String email;
+    private String password;
+    private Boolean isActive;
+
+    @OneToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name = "user_roles")
+    @NonNull
+    private UserRole userRole;
+
+
+    @Override
+    public Collection<? extends GrantedAuthority> getAuthorities() {
+        return null;
+    }
+
+    @Override
+    public String getPassword() {
+        return null;
+    }
+
+    @Override
+    public String getUsername() {
```

Comment:
```	 * Returns the username used to authenticate the user. Cannot return <code>null</code>.```

Reply:
I will change the Spring Security structure (it will be fixed a little bit later).